game/scripts/vscripts/abilities/fountain_attack.lua

```diff
@@ -58,12 +58,20 @@ modifier_fountain_attack_aura = class(ModifierBaseClass)
 
 function modifier_fountain_attack_aura:OnCreated(keys)
   local caster = self:GetCaster()
+  local teamID caster:GetTeamNumber()
   local target = self:GetParent()
-  local attackEffect = "particles/abilities/tesla_coil_radiant.vpcf"
+  if teamID == DOTA_TEAM_GOODGUYS then
+    local attackEffect = "particles/abilities/tesla_coil_radiant.vpcf"
```

Comment:
Um. Also just realised that this is scoped to the `if` block. The variable needs to be declared outside.